### PR TITLE
[raylib] Fix miss header files

### LINKED ISSUE #22935: [New port request] easings/raygui/physac
Library name: raylib

New version number: previous it is 4.0.0#2

It seems that the current raylib package in vcpkg (version 4.0.0#2) misses something with raylib extras, like physac.h, raygui.h, etc. Those headers are included in "extras/... .h" folder and I've checked vcpkg include and there's no "extras", only raylib.h, raymath.h, rlgl.h and probably raudio.h. I think it should be also included into the include directory which is origin under the raylib / src / extras.

### PR DESCRIPTION
**Pull request by JonLiu1993**
Fix #22935 

fix miss header files

### CODE REVIEW

File: ports/raylib/fix-header-miss.patch [hunk 1]
```diff
@@ -0,0 +1,17 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index c96f4f5..938c93a 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -26,7 +26,11 @@ set(raylib_public_headers
+     rlgl.h
+     raymath.h
+     raudio.h
+-    )
++    extras/easings.h
```
Comment (JackBoosY) [inline ports/raylib/fix-header-miss.patch:10]: This should be an upstream bug, can you please report it to the upstream first?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/raylib/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "raylib",
-  "version-semver": "4.0.0",
-  "port-version": 2,
+  "version-date": "2022-02-08",
```
Comment (JackBoosY) [inline ports/raylib/vcpkg.json:3]: We should use a release instead of a commit.

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/raylib/fix-header-miss.patch (hunk 1 above)
Comment (JonLiu1993) [reply ports/raylib/fix-header-miss.patch:10]: Thanks fore your review, I have reported it to the upstream https://github.com/raysan5/raylib/issues/2335

**Review by JackBoosY — APPROVED** [review]

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
See my comment https://github.com/microsoft/vcpkg/issues/22935#issuecomment-1040965494;

Upstream has clarified that these are _NOT_ part of the raylib distribution and are actually vendored dependencies. Requests to deploy them should be treated as requests for new ports from the original sources.

**Comment by frankhale** [comment]
Very sad that this merge was deleted because it gave us what we wanted in lieu of new ports.

**Pull request closed without merging**